docs/root/api/starting_envoy.rst

```diff
@@ -185,6 +185,23 @@ for further information.
   // Swift
   builder.addStreamIdleTimeoutSeconds(5)
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``addPerTryIdleTimeoutSeconds``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Specifies the length of time a retry (including the initial attempt) should wait without a headers
+or data event before timing out. Defaults to 15 seconds.
```

Comment:
Does this apply to trailers?

Reply:
Yes, this should say "wait for bytes". That would be more comprehensive.

Reply:
@buildbreaker let me fix this in a subsequent PR so we don't have to run CI on this again.